tests/test_verify_base_url.py

```diff
@@ -4,6 +4,8 @@
 
 import pytest
 
+from requests.packages.urllib3.util.retry import Retry
```

Comment:
So if we don't print a custom exception message the tests should just look for the error message request throws I am assuming?